state_machine/applications/system-check/main.py

```diff
@@ -0,0 +1,94 @@
+"""
+Python system check script for PyCubed satellite board
+PyCubed Mini mainboard-v02 for Pocketqube Mission
+* Author(s): Yashika Batra
+"""
+
+from lib import pycubed
+import tests
+import tests.i2c_scan
+import tests.logging_test
+import tests.imu_test
+import tests.radio_test
+import tests.sun_sensor_test
+import tests.coil_test
+import tests.burnwire_test
+
+# initialize hardware_dict and result_dict
+hardware_dict = pycubed._cubesat.hardware
+result_dict = {
+    'SDcard_Logging': ('', False),
+    'IMU_AccStill': ('', False),
+    'IMU_AccMoving': ('', False),
+    'IMU_GyroStill': ('', False),
+    'IMU_GyroRotating': ('', False),
+    'IMU_MagMagnet': ('', False),
+    'IMU_Temp': ('', False),
+    'Radio_ReceiveBeacon': ('', False),
+    'Radio_SendBeacon': ('', False),
+    'Sun-Y_Dark': ('', False),
+    'Sun-Y_Light': ('', False),
+    'Sun-Z_Dark': ('', False),
+    'Sun-Z_Light': ('', False),
+    'Sun-X_Dark': ('', False),
+    'Sun-X_Light': ('', False),
+    'Sun+Y_Dark': ('', False),
+    'Sun+Y_Light': ('', False),
+    'Sun+Z_Dark': ('', False),
+    'Sun+Z_Light': ('', False),
+    'Sun+X_Dark': ('', False),
+    'Sun+X_Light': ('', False),
+    'CoilDriverX': ('', False),
+    'CoilDriverY': ('', False),
+    'CoilDriverZ': ('', False),
+    'Burnwire1': ('', False),
+    'Burnwire2': ('', False),
+}
+
+# print acknowledgement that test has started
+print("Running System Check...")
+print("Hardware Successfully Initialized. Printing results...")
```

Comment:
I don't love that this line prints "Successful" every time even if something wasn't successful. Delete `Successfully`.

Let's make this:
```
print("Hardware initialization results:")
```